### ISSUE TITLE
You'll need the OpenLDAP libraries and headers available on your system

### ISSUE THREAD

**Issue by jedie** [django-auth-ldap/django-auth-ldap#217]
This sentence is in the readme and in the documentation:

> You'll need the OpenLDAP libraries and headers available on your system

Can it be explained in more detail? e.g.: What packages need to be installed in a Debian system?

**Comment by francoisfreitag** [comment]
I agree, https://django-auth-ldap.readthedocs.io/en/latest/install.html could be changed to list packages per OS.

**Comment by johnthagen** [comment]
Perhaps it would be good to directly link to `python-ldap`'s install guide? https://www.python-ldap.org/en/python-ldap-3.3.0/installing.html

**Comment by eightnoneone** [comment]
The python-ldap build instructions are helpful, but also carry cruft. They instruct:

`apt-get install build-essential python3-dev python2.7-dev 
    libldap2-dev libsasl2-dev slapd ldap-utils tox
    lcov valgrind`

I found the bare minimum with a python:3.12.8-slim image was:

- `apt-get install build-essential libldap2-dev libsasl2-dev`
- `pip install django-auth-ldap`
